### PR TITLE
Improve how to get `validUntil`

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

File: content/terms/declarations-maintenance.en.md [hunk 1]
```diff
@@ -110,7 +110,16 @@ The consensus is to consider that a service provider renaming a service (for exa
 ## How to get the `validUntil` date from an issue
 
 1. Go to the open issue and scroll to the last comment from `OTA-Bot`.
```
Comment (MattiSG) [inline content/terms/declarations-maintenance.en.md:112]: Without a screenshot, I had to go to https://github.com/OpenTermsArchive/contrib-declarations/pull/1497#discussion_r1654952174 to understand the difference. What about:
```suggestion
1. Go to the open issue and scroll to the last action from `OTA-Bot` (comment or issue opening).
```

and adding a screenshot of the “reopening” case, similar to:

File: content/terms/declarations-maintenance.en.md [hunk 2]
```diff
@@ -109,8 +109,19 @@ The consensus is to consider that a service provider renaming a service (for exa
...
+   ![]({{< relref "." >}}declarations-maintenance-ota-bot-comment.png)
+
+   Note that sometimes:
+
+   - the date is not quite precise, it can be written "last month"
+   ![]({{< relref "." >}}declarations-maintenance-ota-bot-comment-last-month.png)
+   - the last comment may be the announcement of the reopening of the issue
+   ![]({{< relref "." >}}declarations-maintenance-ota-bot-comment-reopened-issue.png)
+
 3. Copy the value of the `datetime` attribute from the `<relative-time>` element.
```
Comment (MattiSG) [inline content/terms/declarations-maintenance.en.md:125]:
```suggestion
3. The value to use as `validUntil` is the value of the `datetime` attribute from the `<relative-time>` element.
```

**Review by MattiSG — APPROVED** [review]

File: content/terms/declarations-maintenance.en.md [hunk 3]
```diff
@@ -109,8 +109,19 @@ The consensus is to consider that a service provider renaming a service (for exa
...
 ## How to get the `validUntil` date from an issue
 
-1. Go to the open issue and scroll to the last comment from `OTA-Bot`.
-2. On the date of this comment, right-click "inspect" to open your browser's code inspector. Note that sometimes, the date is not quite precise, it can be written "last month".
-![]({{< relref "." >}}declarations-maintenance-ota-bot-comment.png)
+1. Go to the open issue and scroll to the last action from `OTA-Bot` (comment or issue opening).
+
+2. On the date of this comment, right-click "inspect" to open your browser's code inspector.
+
+   ![]({{< relref "." >}}declarations-maintenance-ota-bot-comment.png)
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply content/terms/declarations-maintenance.en.md:116]: Yes, I agree!

File: content/terms/declarations-maintenance-ota-bot-comment.png
[the remark this answers is not in the document]
Comment (clementbiron) [reply content/terms/declarations-maintenance-ota-bot-comment.png:1]: I thought I had.

**Pull request merged**